scripts/src/public/routes/orders.ts

```diff
@@ -138,7 +138,7 @@ export type GetOrderHistoryRequest = AuthenticatedRequest & {
 export const getOrderHistory = async function(req: GetOrderHistoryRequest, res: Response) {
 	const filters = {
 		origin: req.query.origin,
-		offerId: req.query.offer_id
+		offerId: req.query.offer_id || req.query.offerId // old clients send offerId instead of offer_id
```

Comment:
This was fixed by in the Android code a while ago, but not all partners have adopted the new version